README.md

```diff
@@ -0,0 +1,31 @@
+
+# Concordium Smart Contract Tools
+
+This repository contains tools to support smart contracts on and off-chain.
+
+- [cargo-concordium](./cargo-concordium) which is a small tool for developing smart contracts. It uses the API exposed in `wasm-chain-integration` to execute smart contracts directly and can initialize and update smart contracts, in a desired state. See the `--help` option of the tool for details on how to invoke it.
+   It can also be used to build contracts embedded with schemas (see section about [contract schemas](#contract-schema)).
+
+
+# Contributing
+
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](https://github.com/Concordium/.github/blob/main/.github/CODE_OF_CONDUCT.md)
```

Comment:
Usually we put the covenant at the top, don't we?

Reply:
I have move the badge a bit up